### ISSUE TITLE
/palm_detection_without_custom_op.tflite ,how to generate?

### ISSUE THREAD

**Issue by wyuzyf** [metalwhale/hand_tracking#4]
thanks for your nice work.
please can you write a README about how to generate this file 
 THANK YOU AGAIN ~

**Comment by shoutOutYangJie** [comment]
Yes, I also want to know how you can import this model without custom op but no error.

**Comment by rover-debug** [comment]
You can use Tensorflow's graph editor to do this.

**Comment by ragavendranbala** [comment]
@bhavints  Can you tell the steps to do that? thanks in advance

**Comment by junhwanjang** [comment]
I reopened my repo:) Please come and feel free to ask!
https://github.com/junhwanjang/mediapipe-models
